docs/modules/ROOT/pages/connect-to-cluster.adoc

```diff
@@ -45,7 +45,7 @@ For a complete tutorial, see the xref:get-started.adoc[] topic.
 [[advanced]]
 == Configure Your Own Client
 
-You can access the information needed to connect your own clients through any of the pre-configured client details. 
+You can access the information needed to connect your own clients through the *Advanced* option. 
```

Comment:
```suggestion
You can access the information needed to connect your own clients through the *Advanced* option. 
```
Is it *Advanced* or *Advanced setup* (as in steps)? Also 'using' rather than 'through'?

Reply:
It is Advanced - that's the name of the option you select.

Yes, it should be using - these changes are for the addition of the Advanced option instead of making people go through one of the pre-configured clients; my attention to detail slipped there! Good spot.

Reply:
I know it's not part of your changes but we should change the step too then - will highlight...

Reply:
Ah, maybe it's tucked up in get-connection-creds.adoc as a partial